compatimetrics.py

```diff
@@ -0,0 +1,501 @@
+    """ Calculates root of mean squared difference of two prediction vectors,
+    which is measure of similarity between two regression models predictions.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+
+    Returns
+    -------
+    Numeric value
+        RMSD value of given predictions pair
+    """
+    return np.sqrt(mse(pred1, pred2))
+
+
+def root_mean_sqaured_error_error_penalty(pred1, pred2, y):
+    """ Calculates root of mean squared difference of two prediction vectors
+    with additional penalty from prediction error considering true values
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+
+    Returns
+    -------
+    Numeric value
+        RMSDEP value of given predictions pair and true values vector
+    """
+    return np.sqrt(mse(pred1, pred2) + mse(pred1, y) / 2 + mse(pred2, y) / 2)
+
+def conjunctive_rmse(pred1, pred2, y):
+    """ Calculates Root Mean Squared Error between real values and prediction
+    being average of two model outputs.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+
+    Returns
+    -------
+    Numeric value
+        Conjunctive RMSE value of given predictions pair and true values vector
+    """
+    pred = []
+    for i in range(len(pred1)):
+        pred.append((pred1[i]+pred2[i])/2)
+    return np.sqrt(mse(pred, y))
+
+
+def strong_disagreement_ratio(pred1, pred2, y):
+    """ Calculates percentage of observations that were predicted strongly
+    different by two models.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    y: list, numpy.array, pandas.series
+        true values of predicted variable
+
+    Returns
+    -------
+    Numeric value
+        SDR value of given predictions pair and true values vector
+    """
+    n = len(pred1)
+    pred1, pred2 = np.array([pred1]), np.array([pred2])
+    difference = np.abs(pred1 - pred2)
+    standard_deviation = np.std(y)
+    count = 0
+    for i in difference[0]:
+        if i > standard_deviation:
+            count += 1
```

Comment:
It seems to me that it can be replaced with `count = np.sum(difference > standard_deviation)`